contributors/scottietremendous.yml

```
@@ -1,2 +1,3 @@
 name: Scott Foerster
 github: scottietremendous
+discord: 'scottfoerster'
```

this needs the numbers too

```suggestion
discord: 'scottfoerster#2844'
```